test/openwhisk.integration.js

```diff
@@ -63,7 +63,7 @@ describe('OpenWhisk Integration Test', () => {
     const out = builder.cfg._logger.output;
     const { auth, namespace } = builder._deployers.wsk._cfg;
     assert.ok(out.indexOf(`ok: 200
-{"url":"/api/v1/web/${namespace}/simple-package/simple-name@1.45.0/foo","file":"Hello, world.\\n"}`) > 0, out);
+{"url":"https://azure.adobe-runtime.com/api/v1/web/${namespace}/simple-package/simple-name@1.45.0/foo","file":"Hello, world.\\n"}`) > 0, out);
```

Comment:
???

Reply:
Runtime changed the output, and now tests are failing.

Reply:
I think the output should remain the same, i.e. use the raw url as defined by the environment

Reply:
> Runtime changed the output, and now tests are failing.

oh!

Reply:
"oh!" is way more polite than what I thought.